core/src/grammar.rs

```diff
@@ -489,6 +489,7 @@ pub enum Statement {
     Transparent {
         body: Vec<Statement>,
     },
+    Empty {},
```

Comment:
I'd put it as the first variant. Since there is no data, there is also no need for braces.

```rust
enum Statement {
    Empty,
    Transparent {
    ...
```